_includes/css/just-the-docs.scss.liquid

```diff
@@ -2,8 +2,8 @@
 $logo: "{{ site.logo | relative_url }}";
 {% endif %}
 @import "./support/support";
-@import "./color_schemes/light";
-@import "./color_schemes/{{ include.color_scheme }}";
+@import "./color_schemes/{{ include.color_scheme | default: 'light' }}";
```

Comment:
Is there an include of `_includes/css/just-the-docs.scss.liquid` somewhere that doesn't set the `color_scheme` parameter to a non-nil value? I couldn't find one.

Reply:
no, because this file has a nil check https://github.com/just-the-docs/just-the-docs/blob/main/assets/css/just-the-docs-default.scss

Reply:
When I run the regression tests with this PR branch, steps 1–3 of the test at https://just-the-docs.github.io/just-the-docs-tests/color/code/ are okay.

But after step 4, I can't rebuild the tests site. The error report is:

```
      Remote Theme: Using theme simonebortolin/just-the-docs
  Conversion error: Jekyll::Converters::Scss encountered an error while converting 'assets/css/just-the-docs-default.scss':
                    Undefined variable: "$link-color". on line 10
jekyll 3.9.2 | Error:  Undefined variable: "$link-color". on line 10
```

I think the problem is that the user may define a color scheme that doesn't define any of the base colors.

Reply:
I need to investigate